### PR TITLE
Some of the general fixups from generator branch

### PR DESCRIPTION
**Pull request by GerryG**
Let me know what other changes in generator that you would want to see here, or in another branch.

Also, please feel free to suggest where we might best add test coverage for this stuff.  I think we need to cover and extend the command line tests, migrations, generators, etc.  Automated or not.  I work on doing the migration from 1.10 test soon.

### CODE REVIEW

File: card/lib/card/cache.rb [hunk 1]
```diff
@@ -25,7 +26,7 @@ class Cache
     class << self
       def [] klass
         raise "nil klass" if klass.nil?
-        cache_by_class[klass] ||= new :class=>klass, :store=>(@@using_rails_cache ? nil : Rails.cache)
+        cache_by_class[klass] ||= new :class=>klass, :store=>(@@using_rails_cache ? nil : Cardio.cache)
```
Comment (ethn) [inline card/lib/card/cache.rb]: you've tried this in a browser, right?

Comment (GerryG) [inline card/lib/card/cache.rb]: In apache/passenger or is wagn s good enough?

**Comment by ethn** [comment]
This looks good to me.  Will merge it in now. Bummer it won't update the diffs on the Generator branch.  Would be much cleaner.

There is this: http://makandracards.com/makandra/13043-force-github-pull-requests-to-update-the-diff-against-its-target-branch

...but I suspect if there aren't comments to preserve, it may be easier to do a new PR.

File: card/lib/card/cache.rb (hunk 1 above)
Comment (ethn) [inline card/lib/card/cache.rb]: just webrick is fine.  when I first looked at card-gem the browser failed on the cache before it loaded the first page.

**Pull request merged**